handlers/smshandler/mongodbstorage.go

```diff
@@ -261,7 +261,9 @@ func (ms *MongoStorage) NewAttempt(userID, electionID types.HexBytes,
 	}
 	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
 	defer cancel()
-	_, err = ms.tokenIndex.InsertOne(ctx, atindex)
+	if r := ms.tokenIndex.FindOne(ctx, bson.M{"_id": token}); r.Err() != nil {
```

Comment:
I would advocate for specifically check that Err is `ErrNoDocuments` as FindOne can return other types of errors and can be useful to act differently depending on the type